From 22a230b1b6a00b8d76f36ee528743a24a069eb58 Mon Sep 17 00:00:00 2001
From: Jenkins slave <kitchen-build@cloudera.com>
Date: Fri, 26 Sep 2014 09:26:29 -0700
Subject: [PATCH 0002/1173] Preparing for CDH5.3.0 development

Conflicts:

	Makefile.vars
	apps/pig/src/pig/tests.py
	desktop/libs/hadoop/java/pom.xml
	maven/pom.xml
---
 Makefile.vars                    |    2 +-
 apps/pig/src/pig/tests.py        |    3 ---
 desktop/libs/hadoop/java/pom.xml |    2 +-
 maven/pom.xml                    |    8 +++++++-
 4 files changed, 9 insertions(+), 6 deletions(-)

diff --git a/Makefile.vars b/Makefile.vars
index 5814991..c3cc8bf 100644
--- a/Makefile.vars
+++ b/Makefile.vars
@@ -82,7 +82,7 @@ ENV_EASY_INSTALL := $(ENV_PYTHON) $(BLD_DIR_BIN)/easy_install
 # This version is substituted through to the tarballs and packages.
 ##############################
 DESKTOP_VERSION := $(shell python <(cat $(ROOT)/VERSION; echo print VERSION))
-MAVEN_VERSION = $(DESKTOP_VERSION)-SNAPSHOT
+MAVEN_VERSION = $(DESKTOP_VERSION)-cdh5.3.0-SNAPSHOT
 
 ##############################
 # Path to the desktop hadoop plugin jar
diff --git a/apps/pig/src/pig/tests.py b/apps/pig/src/pig/tests.py
index 4b4e1f2..3c2ad68 100644
--- a/apps/pig/src/pig/tests.py
+++ b/apps/pig/src/pig/tests.py
@@ -336,9 +336,6 @@ OOZIE_LOGS ="""  Log Type: stdout
   ------------------------
   /var/lib/hadoop-yarn/cache/yarn/nm-local-dir/usercache/romain/appcache/application_1381360805876_0001/container_1381360805876_0001_01_000002
   /etc/hadoop/conf
-  /usr/lib/hadoop/hadoop-nfs-2.1.0-cdh5.0.0-SNAPSHOT.jar
-  /usr/lib/hadoop/hadoop-common-2.1.0-cdh5.0.0-SNAPSHOT.jar
-  /usr/lib/hadoop/hadoop-auth-2.1.0-cdh5.0.0-SNAPSHOT.jar
   /usr/lib/hadoop/hadoop-common.jar
   /var/lib/hadoop-yarn/cache/yarn/nm-local-dir/usercache/romain/appcache/application_1381360805876_0001/container_1381360805876_0001_01_000002/jyson-1.0.2.jar
   ------------------------
diff --git a/desktop/libs/hadoop/java/pom.xml b/desktop/libs/hadoop/java/pom.xml
index 72539b6..b5c1519 100644
--- a/desktop/libs/hadoop/java/pom.xml
+++ b/desktop/libs/hadoop/java/pom.xml
@@ -24,7 +24,7 @@
     <groupId>com.cloudera.hue</groupId>
     <artifactId>hue-parent</artifactId>
     <relativePath>../../../../maven/pom.xml</relativePath>
-    <version>3.7.0-SNAPSHOT</version>
+    <version>3.7.0-cdh5.3.0-SNAPSHOT</version>
   </parent>
 
   <groupId>com.cloudera.hue</groupId>
diff --git a/maven/pom.xml b/maven/pom.xml
index 7af8bc0..1dcfd44 100644
--- a/maven/pom.xml
+++ b/maven/pom.xml
@@ -20,9 +20,15 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <parent>
+    <groupId>com.cloudera.cdh</groupId>
+    <artifactId>cdh-root</artifactId>
+    <version>5.3.0-SNAPSHOT</version>
+  </parent>
+
   <groupId>com.cloudera.hue</groupId>
   <artifactId>hue-parent</artifactId>
-  <version>3.7.0-SNAPSHOT</version>
+  <version>3.7.0-cdh5.3.0-SNAPSHOT</version>
   <packaging>pom</packaging>
 
   <name>Hue Maven Parent POM</name>
-- 
1.7.9.5

